README.md

```diff
@@ -35,59 +36,64 @@ When user back to you page with JWT token, validate it and login user.
 npm i --save silkey-sdk
 ```
 
-### Integration
+### Silkey Integration Guide
 
-#### on signin page
+#### Needed Preperations:
 
-```    
-    const callerParams = {
-      // make sure you support both GET and POST redirection,
-      redirectUrl: 'https://your-registered-domain/redirect-path', 
-      cancelUrl: 'https://your-registered-domain/cancel',
-      refId: 'any',
-      scope: 'email'
-    }
-    
-    const requestParams = await silkeySdk.generateSSORequestParams(callerPrivateKey, callerParams)
+1.  Create a ethereum wallet for the program, save the private key
+2.  Save the application in Apollo with the generated wallet, visit https://apollo.silkey.io to do this
+3.  Store the private key in the .env variables for the application
```

Comment:
lets be more general - there are more ways o store keys in secure way, lets just say  ie: "save key in secure way..."

Reply:
Okay, Ill change that to be more general